src/metabase/models/parameter_card.clj

```diff
@@ -2,13 +2,15 @@
   (:require
    [metabase.util :as u]
    [metabase.util.i18n :refer [tru]]
+   [metabase.util.schema :as su]
+   [schema.core :as s]
    [toucan.db :as db]
    [toucan.models :as models]))
 
 (models/defmodel ParameterCard :parameter_card)
 
 (defonce ^{:doc "Set of valid parameterized_object_type for a ParameterCard"}
-  valid-parameterized_object_type #{"dashboard"})
+  valid-parameterized_object_type #{"dashboard" "card"})
```

Comment:
I understand why it happened, but I'm a little :grimacing: about `-`/`_` mixing and probably would've used hyphens throughout.

Reply:
haha yea I copied the whole word from under. Will change.